src/server/services/email.ts

```diff
@@ -34,10 +32,14 @@ export class MailerNode implements Mailer {
       transporter = nodemailer.createTransport(transporterOptions)
     } else {
       logger.warn(
-        'No SES credentials detected, Using Nodemailer Direct Transport. This should NEVER be seen in production.',
+        'No SES credentials detected, using MailDev at localhost:1080. This should NEVER be seen in production.',
       )
-      // Falls back to direct transport
-      transporter = nodemailer.createTransport(directTransport())
+      // Falls back to maildev
+      transporter = nodemailer.createTransport({
+        host: 'maildev',
+        port: '25',
+        ignoreTLS: true,
+      })
```

Comment:
I realise that I favoured simplicity the last time we discussed this, but.. I think it might be helpful if we were to remove this altogeher and let `config` determine the kind of transport options to feed to MailerNode (see other comments). MailerNode is then focused on one thing - the sending of mails. 

Reading the class more widely, I think we should have a separate PR that focuses on rewriting MailerNode and how it receives its dependencies.